app/modals/stacking/stacking-modal.tsx

```diff
@@ -126,10 +132,12 @@ export const StackingModal: FC<StackingModalProps> = props => {
       burnBlockHeight: coreNodeInfo.burn_block_height,
     });
     const tx = await makeContractCall({ ...txOptions, senderKey: privateKey });
-    const modifiedFeeTx = stackingClient.modifyLockTxFee({
-      tx,
-      amountMicroStx: new BN(amountToStack.toString()),
-    });
+    const modifiedFeeTx = shouldModifyTxFee
```

Comment:
This `modifyLockTxFee` is pretty strange. Looks like it should be replaced with a function like `modifyStackStxTxAmount(amountToStack)`, and the tx fee should be modified with some generic function that works on any tx type.

Reply:
I agree, this method is quite confusing. This is fixed for now though, so will action.